scripts/tests/test_mount_write_encrypted.sh

```
@@ -0,0 +1,87 @@
+
+# Create encrypted archive with one file (use relative path for predictable entry name)
+(cd "$WORKDIR" && echo "original" > seed.txt && \
+  "$PNA_BIN" create "$ARCHIVE" --overwrite --password "$PASSWORD" seed.txt 2>/dev/null || \
+  "$PNA_BIN" create --file "$ARCHIVE" --overwrite --password "$PASSWORD" seed.txt)
+rm -f "$WORKDIR/seed.txt"
+
+mount_enc() {
+  mkdir -p "$MOUNTPOINT"
+  "$PNAFS_BIN" mount --write --password "$PASSWORD" "$ARCHIVE" "$MOUNTPOINT" &
+  MOUNT_PID=$!
+  for i in $(seq 1 10); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)